### ISSUE TITLE
the honkers railway launcher can't choose installation pathway!

### ISSUE THREAD

**Issue by Eigis91** [an-anime-team/the-honkers-railway-launcher#140]
![IMG_20240511_105539](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/169503150/a9c54393-e104-47ca-ad0b-0ba8c252b03c)

I've been playing HSR for 5 months now on my Steam Deck and everything was fine. I have to reformat my Steam Deck cause of the Sudo password issue but that resolved after that. But when I try to reinstalled back, I can't change the pathway installation to my SD card. I'm not the only one with this issue cause others have experience this as well. Could you guys please fix it? I really need to continue playing HSR on my Steam Deck. Thank you!

**Comment by Windmillsarelife** [comment]
Yeah, experiencing the same exact problem too. Can't play at all unfortunately 😔 I hope a quick fix is underway

**Comment by ThunderGemios10** [comment]
Seems to be an issue with the recent release. I managed to get it working by downgrading to an older version (1.5.4) using this command:

`flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`

**Comment by Windmillsarelife** [comment]
> Seems to be an issue with the recent release. I managed to get it working by downgrading to an older version (1.5.4) using this command:
> 
> `flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`

The command isn't working for me :(

After inputting and entering when it wants permission "Y" it gives me an error unfortunately.

**Comment by ThunderGemios10** [comment]
> > Seems to be an issue with the recent release. I managed to get it working by downgrading to an older version (1.5.4) using this command:
> > `flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`
> 
> The command isn't working for me :(
> 
> After inputting and entering when it wants permission "Y" it gives me an error unfortunately.

Try running with `sudo`.

E.g. `sudo flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`

**Comment by Windmillsarelife** [comment]
> > > Seems to be an issue with the recent release. I managed to get it working by downgrading to an older version (1.5.4) using this command:
> > > `flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`
> > 
> > 
> > The command isn't working for me :(
> > After inputting and entering when it wants permission "Y" it gives me an error unfortunately.
> 
> Try running with `sudo`.
> 
> E.g. `sudo flatpak update --commit=754195a3a085af1472edb9fc13c0e9b145dd2dbbd4c700c105502445a6c9f99c moe.launcher.the-honkers-railway-launcher`

I figured it out. Adding the sudo didn't fix it so I just uninstalled and reinstalled and it's downloading properly now.

Thanks for the assistance regardless.

**Comment by Eigis91** [comment]
@ThunderGemios10 Thanks it works! Now I can reinstall my HSR into my SSD!

**Comment by Eigis91** [comment]
> @ThunderGemios10 Thanks it works! Now I can reinstall my HSR into my SSD!

Nope it doesn't work again...

**Comment by stepan323446** [comment]
I also can't change the path in the launcher.

I've tried Flatpak and also Lutris. Through flatpak the launcher did not display the folder selection window at all, and through Lutris it did not change anything

Flatpak:
https://github.com/an-anime-team/the-honkers-railway-launcher/assets/66179087/d8376e49-08e7-41ad-b514-9c3899d15881

Lutris:
https://github.com/an-anime-team/the-honkers-railway-launcher/assets/66179087/f2732e44-baea-4b4e-80cb-68083b2bb296

**Comment by xnfe22** [comment]
same, cant install it on sd card

unable to find game diff

**Comment by stepan323446** [comment]
@xnfe22 It's already fixed (1.6.1 version)

**Comment by xnfe22** [comment]
> @xnfe22 It's already fixed (1.6.1 version)

im using 1.6.1 and still cant install it on SD card

**Comment by stepan323446** [comment]
@xnfe22 Do you use flatpak version? And You see window for choosing directory?

**Comment by xnfe22** [comment]
> @xnfe22 Do you use flatpak version? And You see window for choosing directory?

i download it on discover store and yes i did see the choosing directory after that it always says "unable to find game diff".

**Comment by stepan323446** [comment]
Does another place work besides the CD card?

**Comment by stepan323446** [comment]
Perhaps the problem is in how you mount the CD card into the device. I have all my games installed on another SSD drive

**Comment by MocaRafee** [comment]
> Perhaps the problem is in how you mount the CD card into the device. I have all my games installed on another SSD drive
Where is your SSD path mounted btw?

**Comment by xnfe22** [comment]
still not working on 1.7.1 version, still cant install on sd card

**Comment by MocaRafee** [comment]
> still not working on 1.7.1 version, still cant install on sd card

I got mine working on HDD after creating directory structure like the launcher expected, in my case

/mnt/hdd/.local/share/honkers-railway-launcher/

and now the game sailing smoothly (just fyi, I installed the launcher not from flatpak, but from aur package)

**Comment by xnfe22** [comment]
pls fix cant install on sd card
